### PR TITLE
Made kSQL validation via REST api

### LINKED ISSUE #2: detect missing streams and tables and recreate
in case the kafka streams and topics do not exist the container must be able to re-create them.

### PR DESCRIPTION
**Pull request by swedchef**
Will resolve #2 and in general make life better.. !

### CODE REVIEW

**Review by jacobcrawford — CHANGES_REQUESTED** [review]
The port of the ksql server should be a part of the input and not hardcoded.

File: Dockerfile [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
 FROM python:3.10.0-slim-bullseye
-RUN pip3 install kafka-python pandas
+RUN pip3 install kafka-python pandas requests --no-cache-dir
 
 # Cpy required files into container
-COPY forecast-import.py gridpoints.csv /
+COPY forecast-import.py ksql-config.json gridpoints.csv /
 
 # Run the application
 CMD ["python3", "-u", "forecast-import.py"]
```
Comment (jacobcrawford) [inline Dockerfile:10]: Add newline at the end of file.

File: ksql-config.json [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+            "Key": false
+        },
+        {
+            "ID": "accumulated_global_radiation",
+            "Text": ["Accumulated short wave radiation (J/m2)"],
+            "Type": "ARRAY<DOUBLE>",
+            "Key": false
+        }
+    ]
+}
```
Comment (jacobcrawford) [inline ksql-config.json:101]: Add newline at the end of file.

**Review by jacobcrawford — CHANGES_REQUESTED** [review]
Almost there! Looks beautiful :)

File: Dockerfile (hunk 1 above)
Comment (swedchef) [reply Dockerfile]: Added the newline - thanks

File: ksql-config.json (hunk 2 above)
Comment (swedchef) [reply ksql-config.json]: Added the newline - thanks

**Review by jacobcrawford — APPROVED** [review]
This is awesome!

**Pull request merged**